waggle-dance-core/src/main/java/com/hotels/bdp/waggledance/mapping/service/impl/StaticDatabaseMappingService.java

```diff
@@ -94,17 +100,14 @@ public StaticDatabaseMappingService(
           @Override
           public List<String> load(String key) throws Exception {
             if (primaryDatabaseMapping != null) {
-              return primaryDatabaseMapping.getClient().get_all_databases();
+              return new StaticDatabaseMappingPanopticOperationHandler()
```

Comment:
Is this the issue? We don't filter here?

Reply:
yes, we don't filter. I just did a filtering job.